src/textures/Texture.js

```diff
@@ -225,6 +185,46 @@ Texture.prototype = Object.assign( Object.create( EventDispatcher.prototype ), {
 
 	},
 
+	_getDataURL: function ( image ) {
+
+		var canvas;
+
+		if ( image instanceof HTMLCanvasElement ) {
+
+			canvas = image;
+
+		} else {
+
+			canvas = document.createElementNS( 'http://www.w3.org/1999/xhtml', 'canvas' );
```

Comment:
Since you're already allowing the use of `ImageData`... 
Would you mind adding modifying this so it will work inside a worker?

I think this should suffice:
```javascript
canvas = (typeof OffscreenCanvas !== 'undefined') 
    ? new OffscreenCanvas( image.width, image.height )
    : document.createElementNS( 'http://www.w3.org/1999/xhtml', 'canvas' );
```

More info: [OffscreenCanvas API docs](https://developer.mozilla.org/en-US/docs/Web/API/OffscreenCanvas)

Reply:
Done. ✅